sched/generators.go

```diff
@@ -0,0 +1,51 @@
+package sched
+
+import (
+	"fmt"
+	"github.com/leanovate/gopter"
+	//"github.com/leanovate/gopter/gen"
+)
+
+func GenJob(id string, numTasks int) Job {
+	jobDef := JobDefinition{
+		JobType: "testJob",
+		Tasks:   make(map[string]TaskDefinition),
+	}
+
+	// Generate tasks
+	for i := 0; i < numTasks; i++ {
+		task := GenTask()
+		taskId := fmt.Sprintf("Task%v", i)
+		jobDef.Tasks[taskId] = task
+	}
+
+	job := Job{
+		Id:  id,
+		Def: jobDef,
+	}
+
+	return job
+}
+
+func GenTask() TaskDefinition {
+	return TaskDefinition{
+		Command: Command{
+			Argv: []string{"cmd1", "cmd2"},
```

Comment:
cool fixed!  We can make these more realistic or random if that'd be useful for worker testing in the future too!